CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.21)
```

Comment:
Not sure if we actually need 3.21 minimum, but the default cmake in MSVC 2019 is 3.20.